src/wallet-user/transaction-protection/LocCreation.tsx

```diff
@@ -34,7 +34,10 @@ export default function LocCreation(props: Props) {
         }
     });
     const [ selectedLegalOfficer, setSelectedLegalOfficer ] = useState<LegalOfficer | undefined>();
-    const legalOfficersWithValidIdentityLoc = locsState?.legalOfficersWithValidIdentityLoc;
+    const legalOfficersWithValidIdentityLoc = useMemo(
+        () => locsState?.legalOfficersWithValidIdentityLoc,
+        [ locsState?.legalOfficersWithValidIdentityLoc ]
+    );
```

Comment:
Not sure that memoization brings anything here...

Reply:
Me neither  - but during my tests I experienced a bug you previously fixed with `useMemo()` (endless re-rendering of the same component, or something like this), fixed with this stuff.